pybandits/base.py

```diff
@@ -91,10 +92,14 @@ class BaseMab(PyBanditsBaseModel, ABC):
         The list of possible actions, and their associated Model.
     strategy: Strategy
         The strategy used to select actions.
+    epsilon: Optional[Float01], defaults to 0
+        The probability of selecting a random action.
     """
 
     actions: Dict[ActionId, Model]
     strategy: Strategy
+    epsilon: Optional[Float01] = None
+    default_action: Optional[ActionId]
```

Comment:
in the code below (all the bandits classes) you set default_action to None. Is it what you want to do also here?

Reply:
In contrast to all child classes, this class is abstract, so there was no use to define default value. WDYT? In fact, I can remove the default from epsilon as well...

Reply:
I agree. Let's remove the default in both as you did now.